charts/opentelemetry-collector/README.md

```diff
@@ -46,15 +48,18 @@ There are two ways to configure collector pipelines, which can be used together
 ### Basic top level configuration with `telemetry` property
 
 *Example*: Disable metrics pipeline and send traces to zipkin exporter:
+
 ```yaml
-telemetry:
-  metrics:
-    enabled: false
-  traces:
-    exporter:
-      type: zipkin
-      config:
-        endpoint: zipkin-all-in-one:14250
+configOverride:
+  exporters:
+    zipkin:
+      endpoint: zipkin-all-in-one:14250
+  service:
+    pipelines:
+      metrics: null
```

Comment:
I've just tried this example, and it appeared to be not working with `mustMergeOverwrite`. `metrics` pipeline still has `receivers` and and `processors`, and collector cannot start. 

I'll think how to resolve it later, for now you can bring back the approach you had before.